2 tests failing on clean 2.2.0 repo

**Describe the bug**
2 tests fail when running `npm test` on a clean clone of the master branch at v2.2.0 94edea2bd325c72b952a4b1d52cd46f46209897f. The relevant output is as follows:
```
  metalsmith-permalinks
    √ should change files even with no pattern
    √ should replace a pattern
    √ should ignore any files with permalink equal to false option
    √ should override path in any files with permalink option
    √ should accepts a shorthand string
    √ should copy relative files to maintain references
    √ should not copy relative files
    √ should copy relative files even with patterns
    √ should copy relative files once per output file
    1) should copy files in sibling folder
    √ should format a date
    √ should format a date with a custom formatter
    √ should match arbitrary metadata
    √ should use slug by defaults
    √ should use custom slug config if specified
    √ should use custom slug function
    √ should accept options for slug module
    √ should accept options for slug module and create paths
    √ should make urls unique
    √ should allow a custom function for unique urls
    √ should replace any backslashes in paths with slashes
    √ should use the resolve path for false values (not root)
    √ should use the resolve path for empty arrays (not root)
    2) should return an error when clashes happen


  22 passing (149ms)
  2 failing

  1) metalsmith-permalinks
       should copy files in sibling folder:

      Uncaught AssertionError [ERR_ASSERTION]: '<img src="image.jpg">\n' == '<img src="post/image.jpg">\n'
      + expected - actual

      -<img src="image.jpg">
      +<img src="post/image.jpg">

      at C:\IDE SSD\Git\permalinks-test-test\node_modules\assert-dir-equal\lib\index.js:35:14
      at Array.forEach (<anonymous>)
      at assertDirEqual (node_modules\assert-dir-equal\lib\index.js:31:11)
      at Metalsmith.use.build.err (test\index.js:185:11)
      at Immediate.<anonymous> (node_modules\co\index.js:52:14)

  2) metalsmith-permalinks
       should return an error when clashes happen:

      Uncaught AssertionError [ERR_ASSERTION]: 'Permalinks: Clash with another target file one-post\\index.html' == 'Permalinks: Clash with another target file one-post/index.html'
      + expected - actual

      -Permalinks: Clash with another target file one-post\index.html
      +Permalinks: Clash with another target file one-post/index.html

      at Metalsmith.use.build.err (test\index.js:244:16)
      at Immediate.<anonymous> (node_modules\co\index.js:52:14)



npm ERR! Test failed.  See above for more details.
```

**To Reproduce**
Steps to reproduce the behaviour:
1. Clone repo at commit 94edea2bd325c72b952a4b1d52cd46f46209897f
2. Install dependencies via `npm install`
3. Run `npm test`

**Expected behaviour**
The expected result is all tests passing in a clean clone of the master branch.

**Environment**
 - Windows 7 Professional SP1
 - Node v10.13.0
 - npm 6.4.1

--- a/lib/index.js
+++ b/lib/index.js
@@ -316,7 +316,7 @@ const plugin = options => {
       if (fam) {
         for (const key in fam) {
           if (Object.prototype.hasOwnProperty.call(fam, key)) {
-            const rel = path.join(ppath, key);
+            const rel = path.posix.join(ppath, key);
             dupes[rel] = fam[key];
             moved[key] = rel;
           }
@@ -324,7 +324,7 @@ const plugin = options => {
       }
 
       // add to path data for use in links in templates
-      data.path = ppath === '.' ? '' : ppath;
+      data.path = ppath === '.' ? '' : ppath.replace(/\\/g, '/');
 
       relink(data, moved);
 

